website/static/js/projectorganizer.js

```diff
@@ -989,7 +989,7 @@ function showLegend() {
     });
     var repr = function (item) {
         return [
-            m('span', {
+            m('span[style="width:18px"]', {
```

Comment:
This is fine, but you can do the same more cleanly by passing style as an object:

```js
m('span', {
    style: {width: 18},
    // ...
})
```